src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    static final int MAX_NUMBER = 100;
```

Comment:
According to the naming conventions for constants in Java, the name of the constant should be in uppercase. Please rename `MAX_NUMBER` to follow this convention.